Brewfile

```diff
@@ -1,2 +1,3 @@
 brew 'postgresql'
 brew 'node'
+brew "nginx"
```

Comment:
Because other if someone installs this fresh the nginx part of th bootstrap won't work.